src/actions/helpers/index.ts

```diff
@@ -0,0 +1,21 @@
+import { BigNumber, BigNumberish } from "ethers";
+import { Registrar } from "../../contracts/types";
+
+export const validateOwner = async (
+  domainId: BigNumberish,
```

Comment:
Use `string` for `domainId` please since it will be more consistent with the rest of this project that uses `string` to represent domain id's